monorepo-advantages.md

```diff
@@ -59,11 +59,11 @@ The fact that this is actually a benefit (to the end user) is one of the reasons
 
-As I said, this is a challenging one to discuss. The benefits to the test suites being separate are self-evident, but only if you have a good grasp of the principles at play. If you don't, then this section will have likely read as a recommendation for monorepos. We want our packages to stand on their own. We want to encourage the developers to build and test things in isolation where they are shielded from incidental complexity. We want to maintain small batch sizes so that we can make steady progress and we are not loaded down by all of the problems associated with large batches. It is always easier to combine than it is to separate. If your testing strategy for a package requires testing through the efferents of that package, you have not truly separated that package. That package does not stand on its own and therefore, you will not get any of the benefits of it standing on its own. Imagine if Rails required that all Rails apps were in the Rails repository so that it could run all of the projects' tests instead of Rails having its own test suite.
+As I said, this is a challenging one to discuss. The benefits to the test suites being separate are self-evident, but only if you have a good grasp of the principles at play. If you don't, then this section will have likely read as a recommendation for monorepos. We want our packages to stand on their own. We want to encourage the developers to build and test things in isolation where they are shielded from incidental complexity. We want to maintain small batch sizes so that we can make steady progress and we are not loaded down by all of the problems associated with large batches. It is always easier to combine than it is to separate. If your testing strategy for a package requires testing through the efferents of that package, you have not truly separated that package. That package does not stand on its own and therefore you will not get any of the benefits of it standing on its own. Imagine if Rails required that all Rails apps were in the Rails repository so that it could run all of the projects' tests instead of Rails having its own test suite.
```

Comment:
I'm not sure why the diff doesn't highlight the comma being removed after "and therefore", but thought I'd point it out so as not to leave you scratching your head